tests/integration/test_cli_integration.py

```
@@ -0,0 +1,442 @@
+        mock_pr.body = "This is a test PR"
+        mock_pr.state = "open"
+        mock_pr.user.login = "testuser"
+        mock_pr.created_at = "2024-01-01T00:00:00Z"
+        mock_pr.labels = []
+        mock_pr.assignees = []
+        mock_pr.requested_reviewers = []
+        mock_pr.milestone = None
+        mock_pr.get_reviews.return_value = []
+        mock_pr.get_commits.return_value = []
+
+        mock_repo.get_pull.return_value = mock_pr
```

Ruff (0.13.1)

Undefined name `mock_repo`

(F821)